build/repo.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <AutobahnReportDir>$(ArtifactsDir)autobahnreports\</AutobahnReportDir>
+  </PropertyGroup>
+
+  <Target Name="SetReportDirectory" BeforeTargets="Test">
```

Comment:
The only place I could see this being used was in the test run. Shouldn't matter that it's here instead.